requirements.txt

```diff
@@ -56,7 +56,7 @@ user-agents==1.1.0
 python-geoip-geolite2==2015.303
 chromelogger==0.4.3
 pypd==1.1.0
-disposable-email-domains
+disposable-email-domains==0.0.34
```

Comment:
Why set a specific version?

Reply:
To reduce the chance for breakage on an update of that dependency, for the same reason we pin all other packages.

Reply:
I noticed since I've written this patch there was a new version, it's now at 0.0.45.

Reply:
I know why in general we pin dependencies. It just that in this case it's not pinned on purpose. This is a simple list of domains, so no real danger of breakage. Unless they change the list name (which they did, but kept the old names).

I would rather add a test that asserts the list exists and contains domains rather than pin the version and have the hassle of upgrading it.

As a side note, such changes should be in their own Pull Request.

Reply:
Right, I think the original feature that we had this commit in was for some reason attached to this, and we used pyup at the time that informed us about the unpinned dependency.
Also totally right to split this, I think you reminded me a few times now to not conflate different changes in one PR 😬 Rest assured this isn't on purpose but often a result of the mess of unbundling our fork changes :shakes fist at past: